http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/VertxWebClientIT.java

```diff
@@ -62,8 +62,8 @@ public class VertxWebClientIT {
 
     @QuarkusApplication
     static RestService vertx = new RestService()
-            .withProperty("chucknorris.api.domain", () -> wiremock.getURI(Protocol.HTTP).toString())
-            .withProperty("quarkus.opentelemetry.tracer.exporter.otlp.endpoint", jaeger::getCollectorUrl);
+            .withProperty("chucknorris.api.domain", () -> wiremock.getHost() + ":" + wiremock.getPort())
```

Comment:
this is not one to one change - it uses different (no) protocol and test fails due to incorrect URL